### PR TITLE
Fix invalid operation logging in fast integration KEB client

### PR DESCRIPTION
**Pull request by ksputo**
**Description**

Changes proposed in this pull request:

- Fixed invalid log message `Operation undefined finished with state succeeded`

**Related issue(s)**

### CODE REVIEW

File: tests/fast-integration/kyma-environment-broker/helpers.js [hunk 1]
```diff
@@ -102,11 +102,11 @@ async function ensureOperationSucceeded(keb, kcp, instanceID, operationID, timeo
 
   if (res.state !== 'succeeded') {
     const runtimeStatus = await kcp.getRuntimeStatusOperations(instanceID);
-    throw new Error(`operation didn't succeed in time: ${JSON.stringify(res, null, `\t`)}
+    throw new Error(`operation didn't succeed in time: ${JSON.stringify(res, null, `\t`)}\n
```
Comment (wozniakjan) [inline tests/fast-integration/kyma-environment-broker/helpers.js:105]: nit: is the newline character `\n` at the end desired? I think the fact this is a multiline string will print it on multiple lines.

**Review by wozniakjan — APPROVED** [review]
/lgtm
/hold

feel free to unhold at your convenience

**Review by wozniakjan — APPROVED** [review]

File: tests/fast-integration/kyma-environment-broker/helpers.js (hunk 1 above)
Comment (ksputo) [reply tests/fast-integration/kyma-environment-broker/helpers.js:105]: in line `106` below, the `runtimeStatus` is also printed - that's why the newline character is there

**Pull request merged**